js/components/dataset/resource/form.vue

```diff
@@ -264,7 +265,7 @@ export default {
             const route_new = this.resource.id ? '' : 'new_';
             const route_namespace = this.is_community ? 'community_' : 'dataset_';
             const endpoint = `upload_${route_new}${route_namespace}resource`;
-            return operations[endpoint].urlify(params);
+            return operations && operations[endpoint].urlify(params);
```

Comment:
you're testing for `!operations` above, why do you need it again here?

Reply:
u're right, no need for that anymore.... the blocking point was the missing check against `this.isUpload` upper